docs/src/main/sphinx/connector/hive.rst

```diff
@@ -384,6 +384,11 @@ Property Name                                      Description
                                                    for tables. It must be either 'skip' or an octal number,
                                                    with a leading 0. If set to 'skip', permissions of newly
                                                    created directories will not be set by Trino.
+
+``hive.query-partition-filter-required``           Requires a filter on a partition key for a query.             ``true``
```

Comment:
This sentence does not capture what the property enables/disables. What does it do?

Reply:
Do both HAVING and WHERE work as filters?